test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -2673,6 +2675,62 @@ public async Task DotnetRestore_WithMultiTargettingProject_WhenTargetFrameworkIs
             allTargets.Should().Contain(condition);
         }
 
+        [PlatformFact(Platform.Windows)]
+        public async Task DotnetRestore_withTLSCertificateValidationDisabled_DoesnotThrowException()
```

Comment:
opinionated/nitpick:  This file is already nearly 3,000 lines long, so when a customer reports a bug in restore, it's hard to find if the scenario that the customer is reporting is already tested. This test's focus isn't really about restore, but about testing http communication and if the nuget.config setting applied correctly. So, perhaps we could create a new file whose name make it clear it's about integration tests of our http stack, and the test's implementation just happens to use restore.